src/cmd/serial.js

```diff
@@ -39,6 +39,10 @@ const protip = () => {
 
 const timeoutError = 'Serial timed out';
 
+// An LTE device may take up to 18 seconds to power up the modem
+const MODULE_INFO_COMMAND_TIMEOUT = 20000;
+const IDENTIFY_COMMAND_TIMEOUT = 20000;
```

Comment:
did you consider placing these in `./settings.js` ([example](https://github.com/particle-iot/particle-cli/blob/c9f746b5041ad58e788aa42a710a842b61abcb43/settings.js#L46))?

i realize we have a bunch of these scattered throughout this file so what you have is an improvement but i'm curious about your thinking here.

Reply:
These constants are only relevant to the `i` and `s` commands of the serial console, and I'd keep them closer to the code that actually works with those commands